stdlib/builtins.pyi

```diff
@@ -110,6 +110,8 @@ class object:
 
     def __dir__(self) -> Iterable[str]: ...
     def __init_subclass__(cls) -> None: ...
+    @classmethod
+    def __subclasshook__(cls, __subclass: Any) -> bool: ...
```

Comment:
Surely it doesn't make sense to call this method on things that aren't known to be classes?

```suggestion
    def __subclasshook__(cls, __subclass: type) -> bool: ...
```

Reply:
Done